### PR TITLE
Navbar new

### PR DESCRIPTION
**Pull request by shaikhhamza11**
Description:
1)Deleted the previous documentation page as it was not responsive
2) added responsive navigation components with hamburger  and utilities classes
3)Refactored code structure

### CODE REVIEW

File: css/encore.css [hunk 1]
```diff
@@ -1,86 +1,109 @@
...
-  padding-top: 2rem;
-  box-shadow: 1px 5px 5px 2px var(--color-primary);
+.components .left-nav {
+  /* width: 25%;
+  height: calc(100vh);
+  position: fixed;
+  top: 0;
+  left: 0; */
+}
+
```
Comment (amarphule) [inline css/encore.css:29]: please remove comments

File: css/variables.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    --margin-large: 1.5rem;
+    --margin-extra-large: 2rem; */
+  --container-width: 90%;
+  /* font-size */
+  /* --txt-small: 1rem;
+    --txt-medium: 2rem;
+    --txt-large: 3rem;
+    --txt-ex-large: 4rem; */
+  /* Screen size */
+  /* Mobile(xs),Tablet(sm),Laptop(lg) */
```
Comment (amarphule) [inline css/variables.css]: remove unused commented code

**Comment by tanishk15sharma** [comment]
Follow aram suggestion and you are good to go

**Review by tanishk15sharma — APPROVED** [review]

**Review by shivanigangadharan — APPROVED** [review]
Navbar is well responsive, code also looks good to me.

File: css/encore.css [hunk 3]
```diff
@@ -1 +1,109 @@
...
+.components h1 a {
+  transition: 0.5s;
+  text-decoration: none;
+  padding: 1%;
+  color: #fff;
+  font-weight: 800;
+  font-size: 5rem;
+}
+.components h1 a:hover {
+  /* content: "Click Me!"; */
```
Comment (devananth) [inline css/encore.css:39]: You can remove the unwanted codes ( i.e: commented code )

Comment (shaikhhamza11) [reply css/encore.css:39]: Done

**Pull request merged**